db/migrate/20220831204508_add_bookings_to_pdf.rb

```diff
@@ -0,0 +1,9 @@
+class AddBookingsToPdf < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :pdfs, :booking, null: false, foreign_key: true
+    add_reference :bookings, :offense, null: false, foreign_key: true
```

Comment:
I think you thought you needed this because you had the belongs_to/has_many mixed up but once you fix that you can remove